packages/cli/src/WaitingWebhooks.ts

```diff
@@ -106,18 +76,17 @@ export class WaitingWebhooks {
 			workflow,
 			workflow.getNode(lastNodeExecuted) as INode,
 			additionalData,
-		).find((webhook) => {
-			return (
+		).find(
+			(webhook) =>
 				webhook.httpMethod === httpMethod &&
-				webhook.path === path &&
-				webhook.webhookDescription.restartWebhook === true
-			);
-		});
+				webhook.path === executionId &&
+				webhook.webhookDescription.restartWebhook === true,
+		);
 
 		if (webhookData === undefined) {
 			// If no data got found it means that the execution can not be started via a webhook.
 			// Return 404 because we do not want to give any data if the execution exists or not.
-			const errorMessage = `The execution "${executionId}" with webhook suffix path "${path}" is not known.`;
+			const errorMessage = `The execution "${executionId}" with webhook suffix path "${executionId}" is not known.`;
```

Comment:
for waiting webhooks `path` and `executionId` are the same thing, aren't they?

Reply:
Yeah they should be but let's say you call the url like `/webhook-waiting/potato` instead of an ID, it'd be good to know the full path since the ID is expected to be numerical only at this time.